Lib/_collections_abc.py

```diff
@@ -478,8 +478,7 @@ def __getitem__(self, item):
         # then X[int, str] == X[[int, str]].
         param_len = len(self.__parameters__)
         if param_len == 0:
-            raise TypeError(f'There are no type or parameter specification'
-                            f'variables left in {self}')
+            raise TypeError(f'{self} is not a generic class')
```

Comment:
This is for consistency with the typing version right?

Reply:
There was a minor bug in the old error message: missed space between "specification" and "variables".

Also, there is no reason to have different error messages for two implementations, so I changed one of them. I hope I left the better one.

Reply:
Oh I didn't notice the space. Good eye!

The message is copied from Py_GenericAlias for consistency with the rest of the collections.abc https://github.com/python/cpython/blob/main/Objects/genericaliasobject.c#L303

However, keeping it same with the old `typing.Callable` should be fine too.